### ISSUE TITLE
Quest: That's what Friends Are For (Zul'Drak)

### ISSUE THREAD

**Issue by antonius123** [dalaranwow/dalaran-wow#5017]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
You are expected to find 3 missing crusaders in Zul'Drak. One of them, Crusader Dargath is missing. I have asked players that also said he was missing, some claimed he was seen in the sky a few times as well. A server reset or crash seems to bring the NPC back, but over a period of 3 hours, I was unable to find the NPC, at Dargath's Demise in Zul'drak.

**Expected behaviour**: Tell us what should happen instead
The Crusader should be there as according to:
http://www.wowhead.com/npc=29468/crusader-dargath 

You can even see his aura, but the NPC is either not there or invinsible/untargetable(?) for some reason.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @antonius123,
Issue with creature Crusader Dargath should be solved already, I've also added visual elements to the 3 creatures that are objective of the quest mentionet above.

How it looks like now:
![image](https://user-images.githubusercontent.com/6797611/32618210-b7ce4a6a-c57f-11e7-91e9-aaddf7df6ee7.png)
![image](https://user-images.githubusercontent.com/6797611/32618230-c3d62508-c57f-11e7-9ffc-734437f66e48.png)
![image](https://user-images.githubusercontent.com/6797611/32618243-cca5878c-c57f-11e7-8145-90c785ae0284.png)
